plugins/web/active/Web_Application_Fingerprint@OWTF-IG-004.py

```diff
@@ -6,8 +6,11 @@
 
 DESCRIPTION = "Active probing for fingerprint analysis"
```

Comment:
Here also- blank line b/w import statement and docstring. Almost all files have this problem.